tests/syntax_tests/data/parsing/grammar/expressions/regex.res

```diff
@@ -100,10 +100,7 @@ let re = /a[\\-b]/
 // NOTE: not an error under PCRE/PRE:
 //let re = /a[b-]/
 let re = /a[]b/
-let re = /a[/
```

Comment:
The removed lines are invalid regexes in JavaScript. Some of them now fail (e.g. `/a[/`) thanks to the now correct parsing of character classes.